--- libindi-0.9.8/libs/indibase/alignment/CMakeLists.txt.orig	2014-08-25 12:37:15.100484050 +0200
+++ libindi-0.9.8/libs/indibase/alignment/CMakeLists.txt	2014-08-25 12:37:36.873688353 +0200
@@ -62,7 +62,7 @@ set (AlignmentClient_SRCS
 
 add_library(AlignmentClient ${AlignmentClient_SRCS})
 set_target_properties(AlignmentClient PROPERTIES OUTPUT_NAME indiAlignmentClient)
-install(TARGETS AlignmentClient ARCHIVE DESTINATION ${LIB_DESTINATION})
+install(TARGETS AlignmentClient DESTINATION ${LIB_DESTINATION})
 
 ##################################################
 ############ LoaderCLient test program ###########
